Update zk_login.md (#14472)
MystenLabs/sui: doc/src/build/zk_login.md

## Assemble the zkLogin signature and submit the transaction

> First, sign the transaction bytes with the ephemeral private key using the keypair generated previously. This is the same as [traditional KeyPair signing](https://sui-typescript-docs.vercel.app/typescript/cryptography/keypairs). Make sure that the transaction `sender ` is also defined.

Reviewer: ```suggestion
First, sign the transaction bytes with the ephemeral private key using the key pair generated previously. This is the same as [traditional KeyPair signing](https://sui-typescript-docs.vercel.app/typescript/cryptography/keypairs). Make sure that the transaction `sender ` is also defined.
```

});
  ```

## Caching the ephemeral private key and ZK proof

As mentioned before, each ZK proof is tied to an ephemeral key pair. So the proof can be reused to sign any number of transactions until the ephemeral key pair expires, i.e., until the current epoch crosses `maxEpoch`.

> Developers may thus want to cache the ephemeral key pair along with the ZKP for future uses.

Reviewer: ```suggestion
You might want to cache the ephemeral key pair along with the ZKP for future uses.
```
